examples/proxy/record-validation/cluster-ip/README.md

```diff
@@ -0,0 +1,69 @@
+# Streams for Apache Kafka Proxy Record Validation, exposed using Cluster-IP
+
+In this example, an instance of Apache Kafka is deployed to an OpenShift cluster using Streams for Apache Kafka, alongside Apicurio Registry, which is also deployed to the cluster.
+
+The Streams for Apache Kafka Proxy is deployed with configuration to perform record validation.  The configuration ensures that 
+any records sent to a topic called `people` adhere to a `person` schema.
+
+Finally, Kafka command line tools are used to send valid and invalid records to the `people` topic, so the effects of the validation can be observed.
+so that the effects of the validation can be observed.
+
+# Prerequisites
+
+* Administrative access to the OpenShift Cluster being used to evaluate Streams for Apache Kafka Proxy
+* Streams for Apache Kafka Operator (installed cluster wide)
+* Red Hat build of Apicurio Registry Operator (installed cluster wide)
+* OpenShift CLI (`oc`)
+* `cURL`
+* Apache Kafka CLI tools (`kafka-topics.sh`, `kafka-console-producer.sh`, and `kafka-console-consumer.sh`) found in the `bin` directory of the Streams for Apache Kafka on RHEL distribution
+
+# Deploying the Example
+
+1. Deploy the Example:
+   ```sh
+   oc apply -k cluster-ip
+   oc wait deployment/proxy --for=condition=Available=True --timeout=300s -n kafka-proxy
+   ```
+
+2. Create the example schema in the registry:
+
+   ```sh
+   REGISTRY_URL=http://$(oc get apicurioregistries.registry.apicur.io -n schema-registry registry --template='{{.status.info.host}}')
+   curl -X POST ${REGISTRY_URL}/apis/registry/v2/groups/default/artifacts -H "Content-Type: application/json; artifactType=JSON" -H "X-Registry-ArtifactId: Person" --data @schemas/person.schema.json
```

Comment:
I had an issue with the REGISTRY_URL variable. The value taken by the command is `http://registry.schema-registry/` but I have to set in the curl the whole url pointing to the cluster (no longer alive):
```
http://registry.schema-registry.router-default.apps.<cluster-name>.p1.openshiftapps.com/
```